src/client/java/teammates/client/scripts/DataMigrationHelper.java

```diff
@@ -0,0 +1,34 @@
+package teammates.client.scripts;
+
+public final class DataMigrationHelper {
+
+    private static final int PRINT_CYCLE = 100;
+
+    private DataMigrationHelper() {
+        // utility class
+    }
+
+    /**
+     * Returns true if string contains no characters to be sanitized and contains substrings of sanitized characters
+     */
+    public static boolean isSanitizedHtml(String string) {
+        if (string == null) {
+            return false;
+        }
+        if (string.indexOf('<') >= 0 || string.indexOf('>') >= 0 || string.indexOf('\"') >= 0
+                || string.indexOf('/') >= 0 || string.indexOf('\'') >= 0) {
+            return false;
+        }
+        return string.indexOf("&lt;") >= 0 || string.indexOf("&gt;") >= 0 || string.indexOf("&quot;") >= 0
+               || string.indexOf("&#x2f;") >= 0 || string.indexOf("&#39;") >= 0 || string.indexOf("&amp;") >= 0;
+    }
+
+    /**
+     * Prints the count on system output when count is a multiple of PRINT_CYCLE
+     */
+    public static void printCountRegularly(int count, String itemName) {
+        if (count % PRINT_CYCLE == 0) {
+            System.out.println("On the " + count + "th " + itemName + ".");
+        }
+    }
```

Comment:
Since you are putting this in a helper class, consider extracting a small utility class that can be used in this manner:

```java
LoopHelper loopHelper = new LoopHelper(100, "admin email");

for (...){
    loopHelper.recordLoop(); 
    // other stuff 
}

print("Total count " + loopHelper.total());
```
i.e. the loopHelper object will keep count and print the message at specified intervals. It can be used anywhere else we need some verbosity at regular intervals.